.github/component_owners.yml

```diff
@@ -0,0 +1,29 @@
+# Keep all in alphabetical order
+components:
+  libs/hooks/open-telemetry:
+    - beeme1mr
+    - toddbaert
+  libs/providers/config-cat:
+    - lukas-reining
+  libs/providers/env-var:
+    - beeme1mr
+    - toddbaert
+  libs/providers/flagd:
+    - beeme1mr
+    - toddbaert
+  libs/providers/flagd-web:
+    - beeme1mr
+    - toddbaert
+  libs/providers/go-feature-flag:
+    - thomaspoignant
+  libs/providers/in-memory:
+    - moredip
+    - beeme1mr
+    - toddbaert
+  libs/providers/launchdarkly-client:
+    - kinyoklion
+    - mateoc
+    - sago2k8
+
+ignored-authors:
+  - renovate-bot
```

Comment:
I wasn't quite sure about this.

I guess we can always directly ping authors for renovate PRs that break or stay open for a long time.